features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1913,6 +1913,8 @@
     -->
     <EnableFederatedUserAssociation>{{user.association.enable_for_federated_users}}</EnableFederatedUserAssociation>
 
+    <EnableTenantQualifiedUrls>{{tenant.url_support_enable}}</EnableTenantQualifiedUrls>
```

Comment:
I think this config should change. Can we check with @pulasthi7

Reply:
@pulasthi7 suggested {{tenant_context.qualify_urls}} for the config. WDYT?

Reply:
But it's still missing with which we qualify the URL right. Shouldn't we have that info

Reply:
How about {{tenant_context.qualify_urls_with_tenant}} ?

Reply:
+1